website/content/partials/builder/amazon/common/BlockDevice.mdx

```diff
@@ -20,7 +20,7 @@ launch_block_device_mappings {
 
 JSON example:
 ```json
-launch_block_device_mappings: [
+"launch_block_device_mappings": [
```

Comment:
@fshields this partial is auto-generated from the comments in https://github.com/hashicorp/packer/blob/a906a1b4265bc181c91aa9cf88b9cd3d1d5db5c5/builder/amazon/common/block_device.go#L45

If you make the update to that file and run `make generate` after it will update the documentation in this file. 

Are you able to make the update? If not, I'll be happy to push the change for you if you're okay with me pushing to your branch.